src/OpenTelemetry.AutoInstrumentation/Configurations/TracerSettings.cs

```diff
@@ -89,32 +89,47 @@ protected override void OnLoad(Configuration configuration)
         InstrumentationOptions = new InstrumentationOptions(configuration);
     }
 
-    private static TracesExporter ParseTracesExporter(Configuration configuration)
+    private static IReadOnlyList<TracesExporter> ParseTracesExporter(Configuration configuration)
     {
-        var tracesExporterEnvVar = configuration.GetString(ConfigurationKeys.Traces.Exporter)
-            ?? Constants.ConfigurationValues.Exporters.Otlp;
+        var tracesExporterEnvVar = configuration.GetString(ConfigurationKeys.Traces.Exporter);
 
-        switch (tracesExporterEnvVar)
+        if (string.IsNullOrWhiteSpace(tracesExporterEnvVar))
```

Comment:
My preference would be to use

```suggestion
        if (string.IsNullOrEmpty(tracesExporterEnvVar))
```

it may be good to catch if a user sets some strange whitespaces

Reply:
Reference: https://blog.ploeh.dk/2014/11/18/the-isnullorwhitespace-trap/